src/components/Advertisements/core/AdvertisementEntry/AdvertisementEntry.tsx

```diff
@@ -29,8 +31,8 @@ function advertisementEntry({
 }: InterfaceAddOnEntryProps): JSX.Element {
   const { t } = useTranslation('translation', { keyPrefix: 'advertisement' });
   const [buttonLoading, setButtonLoading] = useState(false);
+  const [dropdown, setDropdown] = useState(false);
   const [showDeleteModal, setShowDeleteModal] = useState(false);
-
```

Comment:
Don't delete this line. It was added to separate state variables from `useMutation` variables.